config.yaml

```diff
@@ -79,3 +79,34 @@ options:
       It is advised to set appropriate number of security-groups and rules.
     type: boolean
     default: false
+  ignore-volume-az:
+    description: |
+    type: boolean
+    default: null
+  bs-version:
+    description: |
+      Used to override automatic version detection. Valid values are v1, v2, v3
+      and auto. When auto is specified automatic detection will select the
+      highest supported version exposed by the underlying OpenStack cloud. If
+      not set, will use the upstream default.
```

Comment:
not sure I understand the upstream default vs auto. Why wouldn't I want the default to be auto?

Reply:
The default of `null` is intended to allow the upstream to provide the default by simply not rendering the config option.  This is to allow for the upstream default to potentially change, which doesn't much apply to this option, but it seemed easier to be consistent.